infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/external/ServiceExposureStrategyProvider.java

```diff
@@ -38,10 +39,22 @@ public ServiceExposureStrategyProvider(
       throw new ConfigurationException(
           format("Unsupported server naming strategy '%s' configured", strategy));
     }
+
+    multiHostStrategy =
+        strategies.get(MultiHostExternalServiceExposureStrategy.MULTI_HOST_STRATEGY);
+    if (multiHostStrategy == null) {
+      throw new ConfigurationException(
+          "No implementation for 'multi-host' server exposure strategy configured even though it is"
+              + " mandatory.");
+    }
   }
 
   @Override
   public ExternalServiceExposureStrategy get() {
     return namingStrategy;
   }
+
+  public ExternalServiceExposureStrategy getMultiHost() {
```

Comment:
`getMultiHostStrategy` maybe? and brief javadoc...

Reply:
:+1: